HelpSource/editor.css

```diff
@@ -6,7 +6,7 @@ textarea.editor {
     height: auto;
     padding: 0 1em;
     margin-top: 1em;
-    border-left: 2px solid #e1e1e1;
+    border-left: 2px solid var(--color-fg-100);
     font-size: 1em;
     line-height: normal;
```

Comment:
and one more thing: could you add 
```css
background: var(--color-cm-background);
```
here please? I know in your themes code background is the same as all the other backgrounds, but it would make sense to see it applied here in case somebody customizes it with two different colors